F4MPBot/cogs/GithubManager.py

```diff
@@ -0,0 +1,85 @@
+from github import Github
+import discord
+import json
+import logging
+import os
+from discord.ext import commands, menus
+
+COG_DIR = os.path.dirname(os.path.abspath(__file__))
+
+GithubBot = Github("Insert Github Personal Access Token (needs read and write permissions for org)")
+
+
+class GithubCommands(commands.Cog):
+    def __init__(self, bot):
+        self.bot = bot
+        self.log = logging.getLogger(__name__)
+
+    @commands.command(brief='<Project Column Name>',
+                      description='This command is used for looking at F4MP Github project tasks. \n Note: May need '
+                                  'to use quotations for multiword args (i.e "Multi Word Arg")')
+    @commands.has_role('Contributor')
+    async def ProjectTasks(self, ctx, args=""):
+        print(args)

```

Comment:
Remove unnecessary print statement, if you wish to log the option to the console, use `logging.DEBUG`.

This also applies to line 26, 33, 40 and 47